org.freecad.FreeCAD.yaml

```diff
@@ -0,0 +1,801 @@
+app-id: org.freecad.FreeCAD
+default-branch: beta
+runtime: org.kde.Platform
+runtime-version: '6.7'
+sdk: org.kde.Sdk
+command: FreeCAD
+desktop-file-name-prefix: "(beta) "
+finish-args:
+  - --share=ipc
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --device=dri
+  - --share=network
+  - --filesystem=host # needed to make file saving work
+  - --filesystem=xdg-run/gvfs # make GnomeVFS accessible
+  - --filesystem=/run/spnav.sock:ro # allow access to spacenavd
+  - --talk-name=org.freedesktop.Flatpak # to launch openscad
```

Comment:
Note to reviewers:

I am the one who implemented this in the original package.

to import openscad file it runs openscad. The wrapper script in there will launch the flatpak of openscad. It has been around for a while.